File: python/token.py
keywords_in_c=['int','float','char','long','double','short','if','else','switch','while','do','for','break','continue','goto','enum','union','struct','auto','static','void','default','case','volatile']
operators=['+','-','*','/','%','&&','||','=']
separators=[',',';','(',')']
input_string=input("enter input string")
tokens=[]
output=''
for char in input_string:
    if char in operators:
        if output!='':
            tokens.append(output)
            output=''
        tokens.append(char)
    elif char in separators:
        if output!='':
            tokens.append(output)
            output=''
        tokens.append(char)
    else:
        if char!=' ':
            output+=char
        else:
            if output in keywords_in_c:
                tokens.append(output)
                output=''

print(tokens)
for token in tokens:
    print(token)

File: java/CD/token.java
package CD;
import java.util.Scanner;

public class token {
    public static void main(String[] args) {
        Scanner input= new Scanner(System.in);
        System.out.println("Enter the String:");
        String str=input.nextLine();
        char[][] token = new char[10][10];
        int j=0,k=0;
        for (int i = 0; i<str.length(); i++) {
            if(str.charAt(i)==' '){
                j++;
                k=0;
            }
            else if (str.charAt(i)=='+'||str.charAt(i)=='-'||
                        str.charAt(i)=='*'||str.charAt(i)=='/'||str.charAt(i)==';'){
                j++;
                token[j][0]=str.charAt(i);
                j++;
                k=0;
            }
            else{
                token[j][k]=str.charAt(i);
                k++;
            }
        }
        System.out.println("TOKENS ARE:");
        for (int i = 0; i<=j; i++) {
            System.out.println(token[i]);
        }
    }
}
